### PR TITLE
Rpc image pruning 160624 1

### PR DESCRIPTION
**Pull request by kylelaffoon**
Relates to #705 
Correct format issues in the following articles:
 - Rackspace Private Cloud - Partner Devices
 - Rackspace Private Cloud Software and sosreport
 - Rackspace Private Cloud Software Package Update - 20 March 2013
 - Rackspace Private Cloud Software - VM Block Migration

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a5867291e6/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-f61f27e550/how-to/).

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-e66898287f/how-to/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-0834973520/how-to/).

File: content/rpc-openstack/rackspace-private-cloud-partner-devices.md [hunk 1]
```diff
@@ -5,13 +5,13 @@ title: Rackspace Private Cloud - Partner Devices
...
-last_modified_date: '2016-01-13'
-last_modified_by: Rose Contreras
+last_modified_date: '2016-06-24'
+last_modified_by: Kyle Laffoon
 product: Rackspace Private Cloud Powered by OpenStack
 product_url: rpc-openstack
 ---
 
-## Rackspace Private Cloud Certified and Supported Devices</strong>
+## Rackspace Private Cloud Certified and Supported Devices
```
Comment (DonNateR) [inline content/rpc-openstack/rackspace-private-cloud-partner-devices.md]: This needs to be switched to an H3 heading

File: content/rpc-openstack/rackspace-private-cloud-software-and-sosreport.md [hunk 2]
```diff
@@ -181,25 +181,25 @@ The following plugin options are available:
...
 Here are some examples of using sosreport plugins to customize the
 report.
 
 -   To enable only the OpenStack plugin:
 
-    ``` {.screen}
+    `` {.screen}
     $ sosreport -o openstack
-    ```
+    ``
```
Comment (DonNateR) [inline content/rpc-openstack/rackspace-private-cloud-software-and-sosreport.md]: I would delete the asterisks and use spacing to make these code blocks work. Same goes for all of the code blocks in this article.

File: content/rpc-openstack/rackspace-private-cloud-software-and-sosreport.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ PPA](https://launchpad.net/~osops-packaging/+archive/ppa).
 The complete list of `sosreport` options is displayed with
 `sosreport --help`.
 
-    ````{.screen}
+    ```{.screen}
```
Comment (DonNateR) [inline content/rpc-openstack/rackspace-private-cloud-software-and-sosreport.md]: Delete asterisks in all code blocks. They appear in the code blocks with the error message. The spacing formats the code blocks correctly already.

File: content/rpc-openstack/rackspace-private-cloud-software-and-sosreport.md [hunk 4]
```diff
@@ -126,80 +124,76 @@ the following example.
...
- nscd        inactive    NSCD related information
- openstack   inactive    OpenStack related information for Debian based
-                         distributions
- radius      inactive    radius related information on Debian distributions
- sar         inactive    Generate the sar file from /var/log/sa/saXX files
- sssd        inactive    sssd-related Diagnostic Information on Debian based
-                         distributions
- xinetd      inactive    xinetd information
+--- | --- | ---
+ as7 | inactive | JBoss related information
```
Comment (DonNateR) [inline content/rpc-openstack/rackspace-private-cloud-software-and-sosreport.md]: Table needs to be fixed

File: content/rpc-openstack/rackspace-private-cloud-software-and-sosreport.md [hunk 5]
```diff
@@ -126,80 +124,76 @@ the following example.
...
- general.syslogsize    15   max size (MiB) to collect per syslog file
- general.all_logs      off  collect all log files defined in syslog.conf
- kernel.modinfo        on   gathers information on all kernel modules
- kvm.topOutput         off  5x iterations of top data
- libraries.ldconfigv   off  the name of each directory as it is scanned,
-                            and any links that are created.
- printing.cups         50   max size (MiB) to collect per cups log file
-```
+--- | --- | ---
+ apache.log | off | gathers all apache logs
```
Comment (DonNateR) [inline content/rpc-openstack/rackspace-private-cloud-software-and-sosreport.md]: Table needs to be fixed.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-29adcb5ea4/how-to/).

**Comment by DonNateR** [comment]
LGTM

**Pull request merged**